frontend/src/routes/+page.svelte

```diff
@@ -1,3 +1,34 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { get } from '$utils/fetch';
+  import { _ } from 'svelte-i18n';
+  import { goto } from '$app/navigation';
+
+  interface Course {
+    name: string;
+    user: {
+      first_name: string;
+    };
+  }
+  let searchQuery = '';
+  let courses: Course[] = [];
+
+  async function getCourses() {
+    const response = await get('/v1/courses');
+    const data = await response.json<Course[]>();
+    courses = data as Course[];
+  }
+
+  function submitForm() {
+    if (searchQuery) {
+      const query = encodeURIComponent(searchQuery.trim());
+      goto(`/courses?search=${query}`);
```

Comment:
Let's use our managed routes here (from `utility/routes`)
You could pass different params there like
```ts
to.courses({search: query})
```

Reply:
Use goto is not relevant?

Reply:
The question was wrong =)